### ISSUE TITLE
[Bug]: Unit test compilation error

### ISSUE THREAD

**Issue by krzys-bfi** [braze-inc/braze-swift-sdk#12]
### Platform

iOS

### Platform Version

iOS 15.5

### Braze SDK Version

5.3.1

### Xcode Version

XCode 13.4.1

### Computer Processor

Intel

### Repro Rate

100%

### Steps To Reproduce

Example:
  1. Add `braze.logCustomEvent(name: "custom_event")` in `AppDelegate.swift`.
  2. Run the app.

### Expected Behavior

Tests building with no issue

### Actual Incorrect Behavior

Compilation errors:

```
Xcode/DerivedData/nightcrawler-agjgeyzlzqsncobyxmswtldetleq/SourcePackages/checkouts/braze-swift-sdk/Sources/BrazeUI/ContentCardUI/ContentCardUISwiftUI.swift:41:47: error: cannot find type 'Context' in scope
    public func makeUIViewController(context: Context) -> BrazeContentCardUI.ViewController {
                                              ^~~~~~~
Xcode/DerivedData/nightcrawler-agjgeyzlzqsncobyxmswtldetleq/SourcePackages/checkouts/braze-swift-sdk/Sources/BrazeUI/ContentCardUI/ContentCardUISwiftUI.swift:50:16: error: cannot find type 'Context' in scope
      context: Context
               ^~~~~~~
Xcode/DerivedData/nightcrawler-agjgeyzlzqsncobyxmswtldetleq/SourcePackages/checkouts/braze-swift-sdk/Sources/BrazeUI/ContentCardUI/ContentCardUISwiftUI.swift:11:35: error: cannot find type 'UIViewControllerRepresentable' in scope
  public struct ContentCardsView: UIViewControllerRepresentable {
```

<img width="745" alt="CleanShot 2022-09-02 at 12 53 53@2x" src="https://user-images.githubusercontent.com/109584150/188125899-3bf82d8e-cc17-48d3-bd4d-d65f0c8b4e17.png">

### Additional Information

- no issue with older AppBoy library
- no issue with compilation without braze swift sdk package

**Comment by lowip** [comment]
Hey @krzys-bfi 

We have just released [`5.3.2`](https://github.com/braze-inc/braze-swift-sdk/releases/tag/5.3.2) which includes a tentative fix for the issue. Would you mind trying it out and report here?

Thanks!

**Comment by krzys-bfi** [comment]
@lowip seems to fix an issue, thanks!
